### ISSUE TITLE
New MucSpinner Component

### ISSUE THREAD

**Issue by matthilo96** [it-at-m/muc-patternlab-vue#608]
**Is your feature request related to a problem? Please describe.**
For ZMS, we need a spinner that contains text in the center.
At the moment, there is only one spinner that can display percentages in the center.

**Describe the solution you'd like**
Add a new component that can display text: MucSpinner.

**Additional context**
The text of the spinner should be read aloud by a screen reader when loading. 
After loading, the screen reader should focus on the last heading.
